VideoLocker/src/main/java/org/edx/mobile/util/observer/AsyncCallableUtils.java

```diff
@@ -0,0 +1,37 @@
+package org.edx.mobile.util.observer;
+
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.Executor;
+
+public enum AsyncCallableUtils {
+    ;
+
+    // Using AsyncTask.THREAD_POOL_EXECUTOR to automatically support Espresso idling resources
+    @VisibleForTesting
+    @NonNull
+    public static Executor EXECUTOR = AsyncTask.THREAD_POOL_EXECUTOR;
```

Comment:
For updating Retrofit or for replacing this class?

Here, I just filed both: https://openedx.atlassian.net/browse/MA-2185

I also commented in the code that it's a temporary interim solution.